src/ucs/sys/sock.c

```diff
@@ -101,14 +101,31 @@ ucs_status_t ucs_socket_setopt(int fd, int level, int optname,
     return UCS_OK;
 }
 
+static const char *ucs_socket_getname_str(int fd, char *str, size_t max_size)
+{
+    struct sockaddr_storage sock_addr = {0};
```

Comment:
no need to initialize to 0

Reply:
it was needed to prevent clang false-positive